app/controllers/users/passwords_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Users::PasswordsController < Devise::PasswordsController
+  # GET /resource/password/new
```

Comment:
@apeksha2697  please remove this file from PR

Reply:
done